R/bambu-processReads.R

```diff
@@ -53,20 +53,49 @@ bambu.processReads <- function(reads, readClass.file, annotations, genomeSequenc
 #' @noRd
 bambu.processReadsByFile <- function(bam.file, genomeSequence, annotations,
```

Comment:
This function seems to have a length of over 50 lines? Maybe can slightly reduce it a bit?

Reply:
the function is now less than 50 lines